CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Refactor output rendering. This removes `config` dependency from `output` package.
```

Comment:
I have omitted CHANGELOG entries in this project, if there are no end-user facing changes, but no strong opinion if this kind of developer facing changes should be included in the CHANGELOG.